README.md

```diff
@@ -5,30 +5,26 @@ It is test scripts of chainer and dockerfiles for its test environment.
 
 ## Requiements
 
+- nvidia-docker
 - Docker (>=1.6)
 - CUDA 7.5
 
 ## How to use
 
-Run `make_docker.py` to make a `Dockerfile` for an environemnt you want to test.
+Run test scripts with options that specify the environment to test.
 
 ```
-$ ./make_docker.py --base=ubuntu14_py2 --numpy=numpy19 --cuda=cuda70 --cudnn=cudnn2
-$ docker build -t chainer_test .
-$ docker run --env COVERALLS_REPO_TOKEN=XXX..X -v ${pwd}:/work -w /work \
-  --device /dev/nvidia0:/dev/nvidia0 \
-  --device /dev/nvidiactl:/dev/nvidiactl \
-  --device /dev/nvidia-uvm:/dev/nvidia-uvm ./test.sh
+$ ./run_install_test.py --base=ubuntu14_py2 --numpy=numpy19 --cuda=cuda70 --cudnn=cudnn2
 ```
 
-## Test scripts
+These test scripts create `Dockerfile`, make containers, and run appropriate test scripts.
 
-- `test.sh`: build and run all tests.
-- `test_cpu.sh`: build and run only tests for CPUs.
-- `test_example.sh`: build and run all examples.
-- `test_example.sh`: build and run all examples of previous version.
-- `test_doc.sh`: build the document and run document test.
-- `test_install.sh`: make source distribution package and install it with given environment.
+
+## Scripts to run
+
+- `run_install_test.py`: Installation test.
+- `run_test.py`: Build and run test scripts.
+- `run_multi_test.py`: Build and run test scripts with .
```

Comment:
`run_test.py` only contains routine tests including py2, py3, py3.5, example, example of previous version and document.
`run_multi_test.py` can run unit tests with all environmental settings such as cuda version, cudnn version, library version, and so on.